examples/classifier.c

```diff
@@ -403,10 +407,33 @@ float** get_predictions (network* net, char** paths, char** labels, int m, int c
 
     srand(time(0));
 
-    for(i = 0; i < m; ++i)
+    // Make array of indices
+    if(max > m) {
+      max = m;
+    }
+    int shuffled_indices[max];
+    for(int i=0; i<max; i++){
+      shuffled_indices[i] = i;
+    }
+    // Shuffle only if not using every sample
+    if(max != m)
+    {
+      // Knuth shuffle
+      for(int i=(max-1); i>0; i--){
+        int random = rand() % i;
```

Comment:
Make this random in the entire range instead

Reply:
corrigido